generator/uscore/uscore_generator.rb

```diff
@@ -1024,9 +1024,9 @@ def get_first_search_with_fixed_values(sequence, search_parameters, save_resourc
           #{skip_if_search_not_supported_code(sequence, search_parameters)}
           @#{sequence[:resource].underscore}_ary = {}
           @resources_found = false
-          #{'values_found = 0' if find_two_values}
           #{values_variable_name} = [#{search_param[:values].map { |val| "'#{val}'" }.join(', ')}]
           patient_ids.each do |patient|
+            #{'values_found = 0' if find_two_values}
```

Comment:
There's only a `break` in the inner `.each`, so even when a patient is found with two status, it continues to perform searches for each other patient. Is that the intention?

![Screen Shot 2020-10-05 at 2 57 01 PM](https://user-images.githubusercontent.com/15969967/95120324-5f574b80-071b-11eb-9d5f-1a3a61e7c439.png)

Reply:
I think this always worked this way, and this doesn't change that. @czh-orz should weigh in.

Reply:
I think in this case, since this is the most 'general' of careteam searches, we shouldn't even be stopping after we find 2 different statuses within the same patient, because we might not find all resources that are relevant for hte 'must support' checks later.  We are communicating (or at least I am), that you must have all the must support elements somewhere across all resources, but we stop prematurely in this case it seems?

Reply:
You're right that this might affect the must support checks. In that case, we shouldn't break once two have been found and we should just loop through all the patients provided.